### PR TITLE
Fix time range for for daily stats

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé

Update the time range used by daily stats in the `fetch_notification_status_for_service_for_today_and_7_previous_days` function to calculate over the time range since the previous midnight UTC.  This is to align with how our daily limits work.

## Test instructions | Instructions pour tester la modification

- Sends before midnight UTC should not be part of the total contained in the `Daily Usage` section

## Release Instructions | Instructions pour le déploiement

- None.

### CODE REVIEW

File: app/dao/fact_notification_status_dao.py [hunk 1]
```diff
@@ -313,6 +319,86 @@ def fetch_notification_status_for_service_for_today_and_7_previous_days(service_
     ).all()
 
 
+def fetch_daily_notification_status_for_service(service_id, by_template=False):
```
Comment (jzbahrai) [inline app/dao/fact_notification_status_dao.py:322]: this is a new function? I don't see where it is called in this PR?

File: app/dao/fact_notification_status_dao.py [hunk 2]
```diff
@@ -313,6 +319,86 @@ def fetch_notification_status_for_service_for_today_and_7_previous_days(service_
...
+        )
+    )
+
+    all_stats_table = stats_for_today.subquery()
+
+    query = db.session.query(
+        *(
+            [
+                Template.name.label("template_name"),
+                Template.is_precompiled_letter,
```
Comment (jzbahrai) [inline app/dao/fact_notification_status_dao.py:356]: I am proabaly missing this but why do we need the line is_precompiled_letter?

File: app/dao/fact_notification_status_dao.py (hunk 1 above)
Comment (andrewleith) [reply app/dao/fact_notification_status_dao.py:322]: Ah my bad, at first I was going to make a new method so I wouldnt affect other things, but I thought better of that.  Let me clean that up.

**Review by jzbahrai — APPROVED** [review]

**Review by whabanks — APPROVED** [review]
LGTM! Let's get this in staging and see how it goes, fingers crossed!

File: app/dao/fact_notification_status_dao.py (hunk 2 above)
Comment (andrewleith) [reply app/dao/fact_notification_status_dao.py]: This is existing code - not sure.

**Pull request merged**